layers/+completion/auto-completion/funcs.el

```diff
@@ -302,10 +302,13 @@ MODE parameter must match the :modes values used in the call to
    ((or (eq 'vim style)
         (and (eq 'hybrid style)
              hybrid-style-enable-hjkl-bindings))
-    (let ((map company-active-map))
+    (let ((map company-active-map) (smap company-search-map))
```

Comment:
this can be replaced by a dolist form

Reply:
Thanks for the tip; I wasn't familiar with the `dolist` macro but I agree it's nicer and produces a much tidier diff. I've force-pushed the amended commit (since per the contributing guidelines I see a single commit per PR is desirable).